src/containers/Home/index.tsx

```diff
@@ -1,7 +1,65 @@
 import React from 'react';
+import useForm, { isRequired, isEmail, maxChars } from '../../hooks/useForm';
 
 const Home: React.FC = () => {
-  return <h2>Home</h2>;
-}
+  const { fields, onChange, errors, disableForm } = useForm([
+    { field: 'name', value: '', validators: [isRequired] },
+    { field: 'email', value: '', validators: [isRequired, isEmail] },
+    { field: 'comment', value: '', validators: [maxChars(20)] },
```

Comment:
This would just be another variable, it is like when we use useState, we can initialize the hook with a value directly or we can pass a variable to the hook. I dont see it necessary to have an extra variable here